### PR TITLE
docs: docs workflow should pick up OpenAPI spec changes

### PR DESCRIPTION
**Pull request by lalithsuresh**

### CODE REVIEW

**Review by mihaibudiu — APPROVED** [review]

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -6,10 +6,12 @@ on:
   pull_request:
     paths:
       - docs/**
+      - crates/pipeline_manager/src/**
```
Comment (gz) [inline .github/workflows/docs.yml:9]: might want to set this to just src/pipeline_manager.rs as all the OpenAPI relevant parts are contained to this file anyway to try and keep CI time sanity

Comment (gz) [reply .github/workflows/docs.yml:9]: nvm I guess this doesn't work as it includes types from other files :(

**Comment by lalithsuresh** [comment]
@gz that's unfortunately not the case, because there are types elsewhere in the code that are being used to generate documentation. E.g., `PipelineConfig.rs` and the other types in that file.

File: .github/workflows/docs.yml (hunk 1 above)
Comment (lalithsuresh) [reply .github/workflows/docs.yml:9]: Yeah exactly :(

**Pull request merged**